etna/transforms/timestamp/holiday.py

```diff
@@ -27,8 +71,15 @@ class HolidayTransform(IrreversibleTransform):
     """
     HolidayTransform generates series that indicates holidays in given dataset.
 
-    In ``binary`` mode shows the presence of holiday in that day. In ``category`` mode shows the name of the holiday
-    with value "NO_HOLIDAY" reserved for days without holidays.
+    - In ``binary`` mode shows the presence of holiday in that day.
+    - In ``category`` mode shows the name of the holiday with value "NO_HOLIDAY" reserved for days without holidays.
+    - In ``days_count`` mode shows the frequency of holidays in a given period.
```

Comment:
It still doesn't render correctly: https://deploy-preview-240--etna-docs.netlify.app/api_reference/api/etna.transforms.holidaytransform#etna.transforms.HolidayTransform
Try to experiment with formatting until it correctly renders list and sublist. Use links in our contribution guide to learn about ReST formatting.